components/service/glean/src/main/java/mozilla/components/service/glean/storages/StorageEngineManager.kt

```diff
@@ -32,6 +33,7 @@ internal class StorageEngineManager(
         "datetime" to DatetimesStorageEngine,
         "events" to EventsStorageEngine,
         "memory_distribution" to MemoryDistributionsStorageEngine,
+        "quantity" to QuantitiesStorageEngine,
```

Comment:
nit: this list _was_ in alphabetic order...

Reply:
and I think it still is, unless I'm missing something...?

Reply:
Wait, what...   I know that was in there in a different order the first time I looked LOL!  Move along, nothing to see here.